[new product]  privatebin (#4657)
captn3m0/endoflife.date: products/privatebin.md

---
title:  PrivateBin
category: server-app

> tags: pastebin encryption privacy-security

Reviewer: ```suggestion
```

See https://endoflife.date/contribute.

permalink: /privatebin

> alternate_urls:
> -   /private-bin

Reviewer: ```suggestion
```

PrivateBin is always referred as `PrivateBin`, I did not found any reference to `Private Bin`.

> #releasePolicyLink: https://support.hashicorp.com/hc/articles/360021185113

Reviewer: ```suggestion
releasePolicyLink: https://github.com/PrivateBin/PrivateBin?tab=security-ov-file#readme
```

> changelogTemplate: https://github.com/hashicorp/vault/releases/tag/__LATEST__

Reviewer: ```suggestion
changelogTemplate: https://github.com/PrivateBin/PrivateBin/releases/tag/__LATEST__
```

releaseDateColumn: true

> #versionCommand: vault --version

Reviewer: ```suggestion
```

Unrelated

auto:
  methods:
  -   git: https://github.com/PrivateBin/PrivateBin.git

> # eol(x) = release(x+3)

Reviewer: ```suggestion
# eol(x) = releaseDate(x+1)
```

According to https://github.com/PrivateBin/PrivateBin?tab=security-ov-file#readme.

releases:
-   releaseCycle: "1.7"
    releaseDate: 2024-02-12
    eol: false

>     latest: "1.7.1"
>     latestReleaseDate: 2024-02-12

Reviewer: ```suggestion
    latest: "1.7.4"
    latestReleaseDate: 2024-07-09
```

-   releaseCycle: "1.6"
    releaseDate: 2023-09-12
    eol: 2024-02-12
    latest: "1.6.2"
    latestReleaseDate: 2023-12-15

-   releaseCycle: "1.5"
    releaseDate: 2022-12-11
    eol: 2023-09-12
    latest: "1.5.2"
    latestReleaseDate: 2023-07-09

-   releaseCycle: "1.4"
    releaseDate: 2022-04-09
    eol: 2022-12-11
    latest: "1.4.0"
    latestReleaseDate: 2022-04-09

-   releaseCycle: "1.3"
    releaseDate: 2019-07-10
    eol: 2022-04-09
    latest: "1.3.5"
    latestReleaseDate: 2021-04-06

-   releaseCycle: "1.2"
    releaseDate: 2018-07-22
    eol: 2019-07-10
    latest: "1.2.3"
    latestReleaseDate: 2020-02-16

-   releaseCycle: "1.1"
    releaseDate: 2016-12-26
    eol: 2018-07-22
    latest: "1.1.1"
    latestReleaseDate: 2017-10-11 

> ---

Reviewer: ```suggestion

-   releaseCycle: "1.0"
    releaseDate: 2016-08-25
    eol: 2016-12-26
    latest: "1.0"
    latestReleaseDate: 2016-08-25

---
```

> > [PrivateBin](https://privatebin.info/) is a minimalist, open source online [pastebin](https://en.wikipedia.org/wiki/Pastebin)
> > where the server has zero knowledge of pasted data.
> > Data is encrypted and decrypted in the browser using 256bit AES in Galois Counter mode.

Reviewer: ```suggestion
> [PrivateBin](https://privatebin.info/) is a minimalist, open source [Pastebin](https://pastebin.com/) where the server
> has zero knowledge of pasted data.
```

Removed technical details that may change in the future.

> A major release is identified by a change in the first (X) or second (Y) digit in the following
> versioning nomenclature: `Version X.Y.Z.`

Reviewer: ```suggestion
PrivateBin follows [SemVer](https://semver.org/). Only the latest version is supported.
```

Shorter, and document the support policy.
